pcbuilder_crawlers/pipelines.py

```diff
@@ -6,19 +5,12 @@
 
 
 class Pipeline(object):
-    def __init__(self):
-        self.root_url = config.get("API_URL")
-        # self.api_url = ("{root_url}category/{category_name}/product"
-        #                 "/{product_id}" .format(root_url=self.root_url))
 
     def process_item(self, item, spider):
         template = TemplateInterface()
         category = template.translate_category(item['category'])
-        if spider is HardwareInfoSpider:
```

Comment:
Why are we removing this? The reason this is being done is that we want to handle the specifications crawler and the others differently. The other webshop crawlers only care about updating prices. They don't care about specifications and because of that there's no reason to waste performance translating them.

For the other webshop crawlers we want the following implementation:
```
post_price(item):
product =  # get the product by making a get request with the ean / sku
product['price'] = item['price']
# post the product
```

Keep in mind this does not implement the price record history yet.